google/ggrc-core
# GGRC-3699 Rename role 'Admin' to 'Admins'

# Issue description

Rename role 'Admin' to 'Admins'.

# Steps to test the changes
1. Log into ggrc app.
2. Create multiple objects having 'Admins' role.
3. Verify the presence of 'Admins' role in their respective info pages, info panes.
4. In objects tree list view, check Column + Set visible fields pop-up 
5. Try filtering objects using expressions like, **Admins="testuser@example.com"** . It should filter objects matching the provided expression.
6. Open export page, select any created object type and check if 'Admins' is visible.
7. Export any object to a spreadsheet and check the spreadsheet for the presence of 'Admins'
8. Modify the exported spreadsheet and import it again. 
9. Import must be successful.
10. Create a Program and an Audit based on it.
11. Map few already created objects to the Audit.
12. Create an Assessment based on Audit created.
13. Check if pop-ups: Unified Mapper, Assessment Generator contain provision to filter objects based on 'Admins'.

# Solution description
* Updated the 'name' column of 'access_control_roles' tables to 'Admins', where 'object_type' is in the below list: OBJECT_TYPES.
OBJECT_TYPES = [
    "AccountBalance",
    "AccessGroup",
    "Contract",
    "Control",
    "DataAsset",
    "Facility",
    "Issue",
    "KeyReport",
    "Market",
    "Metric",
    "Objective",
    "OrgGroup",
    "Policy",
    "Process",
    "Product",
    "ProductGroup",
    "Project",
    "Regulation",
    "Requirement",
    "Risk",
    "Standard",
    "System",
    "TechnologyEnvironment",
    "Threat",
    "Vendor",
    "Workflow",
]

# Sanity checklist




# PR Review checklist
